### PR TITLE
fix: README grammer/wording

### PR DESCRIPTION
**Pull request by stocaaro**
#### Description of changes
Re-reading the README update and making updates to language that reads awkwardly.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,15 +127,15 @@ To get started pick your platform from our [**Getting Started** home page](https
   + }
   ```
 
-- `Analytics.record` no longer supports passing a string only as input. For example:
+- `Analytics.record` no longer supports passing a string only inputs. For example:
```
Comment (abdallahshaban557) [inline README.md:130]: This still reads a bit weird for me. Maybe removing inputs here?

**Review by abdallahshaban557 — COMMENTED** [review]
Left feedback

File: README.md (hunk 1 above)
Comment (jimblanc) [inline README.md:130]: Should be `input` I think (or remove the "a")?

Comment (cshfang) [inline README.md:130]: Suggest we just be concise
```suggestion
- `Analytics.record` no longer accepts string as input. For example:
```

File: README.md [hunk 2]
```diff
@@ -127,15 +127,15 @@ To get started pick your platform from our [**Getting Started** home page](https
...
 
   ```diff
   - Analytics.record('my example event');
   + Analytics.record( { name: 'my example event' });
   ```
 
 - The `JS` export has been removed from `@aws-amplify/core` in favor the exporting the functions it contained.
-- Any calls to `Amplify.Auth`, `Amplify.Cache`, and `Amplify.ServiceWorker` are no longer supported, instead use the named exports. For example:
+- Any calls to `Amplify.Auth`, `Amplify.Cache`, and `Amplify.ServiceWorker` are no longer supported, instead your code should use the named exports. For example:
```
Comment (cshfang) [inline README.md:138]: Grammar nit:
```suggestion
- Any calls to `Amplify.Auth`, `Amplify.Cache`, and `Amplify.ServiceWorker` are no longer supported. Instead , your code should use the named exports. For example:
```

File: README.md [hunk 3]
```diff
@@ -127,15 +127,15 @@ To get started pick your platform from our [**Getting Started** home page](https
   + }
   ```
 
-- `Analytics.record` no longer supports passing a string only as input. For example:
+- `Analytics.record` no longer supports passing a string only inputs. For example:
 
   ```diff
   - Analytics.record('my example event');
   + Analytics.record( { name: 'my example event' });
```
Comment (cshfang) [inline README.md:134]: Super nit while we're at it to raise the bar..
```suggestion
  + Analytics.record({ name: 'my example event' });
```

File: README.md [hunk 4]
```diff
@@ -127,15 +127,15 @@ To get started pick your platform from our [**Getting Started** home page](https
...
 
   ```diff
   - Analytics.record('my example event');
-  + Analytics.record( { name: 'my example event' });
+  + Analytics.record({ name: 'my example event' });
   ```
 
 - The `JS` export has been removed from `@aws-amplify/core` in favor the exporting the functions it contained.
-- Any calls to `Amplify.Auth`, `Amplify.Cache`, and `Amplify.ServiceWorker` are no longer supported, instead your code should use the named exports. For example:
+- Any calls to `Amplify.Auth`, `Amplify.Cache`, and `Amplify.ServiceWorker` are no longer supported. Instead , your code should use the named exports. For example:
```
Comment (jimblanc) [inline README.md:138]: Nit: Extra space before comma `Instead ,`

**Review by jimblanc — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (cshfang) [reply README.md:138]: Oops

File: README.md [hunk 5]
```diff
@@ -127,15 +127,15 @@ To get started pick your platform from our [**Getting Started** home page](https
...
 
   ```diff
   - Analytics.record('my example event');
-  + Analytics.record( { name: 'my example event' });
+  + Analytics.record({ name: 'my example event' });
   ```
 
 - The `JS` export has been removed from `@aws-amplify/core` in favor the exporting the functions it contained.
-- Any calls to `Amplify.Auth`, `Amplify.Cache`, and `Amplify.ServiceWorker` are no longer supported, instead use the named exports. For example:
+- Any calls to `Amplify.Auth`, `Amplify.Cache`, and `Amplify.ServiceWorker` are no longer supported. Instead , your code should use the named exports. For example:
```
Comment (cshfang) [inline README.md:138]: Remove extra space
```suggestion
- Any calls to `Amplify.Auth`, `Amplify.Cache`, and `Amplify.ServiceWorker` are no longer supported. Instead, your code should use the named exports. For example:
```

**Review by cshfang — APPROVED** [review]
Just a quick nit fix for my nit fix

File: README.md (hunk 5 above)
Comment (stocaaro) [reply README.md:138]: Well garbage. I scrolled past this and hit submit before seeing it :-/  Guess I get to pad my commit stats a little bit more... https://github.com/aws-amplify/amplify-js/pull/10670

**Pull request merged**